en-US/R.xml

```diff
@@ -118,7 +118,8 @@
 			<term>refer to</term>
 			 <listitem>
 				<para>
-					Do not use to indicate a reference (within a manual) or a cross-reference (to another manual or documentation source). Use "See".
+					Use to refer readers to another resource. 
```

Comment:
For clarity, I suggest not using the term itself in the description. Suggest: Use to provide readers with a resource.